modules/flake-parts/nixosConfigurations.linux-builder-01/configuration.nix

```diff
@@ -74,7 +74,7 @@
   };
 
   sops.secrets.github-runners-token = {
-    key = "gh_hra2_pat3";
```

Comment:
PAT was expired. Is it used anywhere else?

Reply:
rotation reminders would really be great. one low-tech way to enable this would be to add the key expiration as comments, however they're [apparently always encrypted](https://github.com/getsops/sops/issues/921) so a reminder job would need secrets access which seems too risky.